web/src/App.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable import/no-named-as-default */
+import { Route, Switch, BrowserRouter as Router } from 'react-router-dom'
+import React, { useState } from 'react'
+import { hot } from 'react-hot-loader'
+import { StoreProvider } from './hooks/useStore'
+import HomePage from './ui/pages/HomePage/HomePage'
+import Menu from './ui/components/Header/Menu'
+import Modal from './ui/components/Modal/Modal'
+import store from './utils/store'
+import computeLayoutConfig from './utils/computeLayoutConfig'
+
+// Import Tabler styles
+import './assets/scss/tabler.scss'
+
+import './App.scss'
+
+const App = () => {
+  const [showAuthModal, setShowAuthModal] = useState(!store.getItem('auth_token'))
+  const [authToken, setAuthToken] = useState(store.getItem('auth_token') || '')
+  const searchParams = new URLSearchParams(window.location.search)
+  const urlData = {
+    targets: searchParams.getAll('targets').join(',') || '',
+    withClosed: searchParams.get('withClosed') || '',
+    withoutIsolated: searchParams.get('withoutIsolated') || '',
+    withoutPrs: searchParams.get('withoutPrs') || '',
+    withoutExternalDeps: searchParams.get('withoutExternalDeps') || '',
```

Comment:
be careful with the variable prefixes

"withXXX" should be true if the checkbox is checked
"withoutXXX" shoule be false if the checkbox is checked